packages/shared-ux/page/analytics_no_data/src/mocks.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import {
+  getKibanaNoDataPageStoryArgTypes,
+  getKibanaNoDataPageStoryMock,
+  getKibanaNoDataPageJestServices,
+} from '@kbn/shared-ux-page-kibana-no-data';
+
+import { AnalyticsNoDataPageServices } from './services';
+
+export type Params = Record<keyof ReturnType<typeof getStoryArgTypes>, any>;
+
+export const getStoryServices = (params: Params) => {
+  // Solution and Logo do not apply, as they are defaulted in this component;
+  const services: AnalyticsNoDataPageServices = {
+    ...getKibanaNoDataPageStoryMock({ ...params, solution: 'Analytics', logo: 'logoKibana' }),
+    kibanaGuideDocLink: 'Kibana guide',
+  };
+
+  return services;
+};
+
+export const getStoryArgTypes = () => {
+  // Solution and Logo do not apply, as they are defaulted in this component;
+  const { solution, logo, ...args } = getKibanaNoDataPageStoryArgTypes();
```

Comment:
I love this way of composing arguments, so we can reuse them. 
At the same time, I feel we are reducing the readability of the story itself, as it's not immediately obvious what the args are. Even the comment itself feels misplaced - "they are defaulted in *this* component". Where is `this` we're referring to? Should we make this a part of the `story` file?

Reply:
I should clarify this comment:

`solution` and `logo` are props on the `KibanaNoDataPage`, and are therefore set by `AnalyticsNoDataPage` internally... therefore `getStoryArgTypes` needs to not include them for any story for `KibanaNoDataPage`.